CHANGELOG.md

```diff
@@ -16,12 +16,38 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 ## Unreleased
 
 ### Added
+- Transitioned Capital Framework dependency to v1.0.0 in bower.json.
+- Added gulp and the required npm plugins
+- Added gulp config file to lay out configs for each task
+- Added gulp tasks split up into their own files
+- Added acceptance tests for `/offices/*` pages accessible through site's menu.
+- Added Accessibility page to footer and adds Accessibility page tests.
+- Added acceptance tests for `/sub-pages/*`.
+- Added `activities-block` shared template for activity feed
+  on offices and sub-pages.
+- Added accessibility complaint form.
+- Added "File an EEO Issue" form.
 
 ### Changed
+- Site's "About" text to "About Us".
+- Replaced FOIA Records with Coming Soon heading
+- Updated setup.sh to use gulp
+- Updated travis to use gulp tasks
+- Updated main.less to use the paths option in less compiler.
+- Moved and renamed contact-macro to contact-layout in macros directory.
+- Moved filters macro from `post-macros.html` to `/macros/filter.html`.
+- Makes filters macro helpers private.
 
 ### Removed
+- Removed Grunt plugins from package.json
+- Removed the Gruntfile.
+- Removed homepage progress charts and related content and JS.
+- Removed 80px to 120px sizing for the isocon sizes on the-bureau page.
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/e21d9d1dda625d812cfd684cdff6e498eafd0506